content/2-how-crs-works/2-3-false-positives-and-tuning.md

```diff
@@ -241,6 +246,11 @@ SecRuleRemoveByTag attack-sqli
 SecRuleUpdateTargetById 941320 "!ARGS:wp_post"
 ```
 
+{{% notice %}}
+If, for optimization purposes, you need to use dynamic variables - for example, to efficiently filter Base64 variants - be aware that this directive supports dynamic variables, like so:
```

Comment:
This notice is more confusing than helpful, I think. It's unclear to the reader, where the variable `base64_ARGS` comes from. I like the example though, so maybe transform this into a separate scenario, and include setting `base64_ARGS`.